### PR TITLE
chore(CPlusPlus): added math factorial

### PR DESCRIPTION
**Pull request by samughodake**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by samughodake** [comment]
Hi @ming-tsai,
No one has reviewed this pull request yet.
Will you please assign a different reviewer to this?
Thanks.

File: algorithms/CPlusPlus/Maths/factorial.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return f;
+}
+
+//main starts
+int main() {
+  cout << "Enter a number: \n";
+  long n;
+  cin>>n;
+  cout<<"factorial is: "<<factorial(n);
+}
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Maths/factorial.cpp:32]: and return 0 and the end of the main function to tell that your program runs successfully

File: algorithms/CPlusPlus/Maths/missing-number.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+cout << "Enter number of elements:\n";
+int n;
+cin>>n;
+vector<int> v(n);
+cout<<"Enter any "<<n<<" elements (From 0 to "<<n<<") \n";
+for(int i=0;i<n;i++){
+  cin>>v[i];
+}
+cout<<"\nMissing number is: "<<missingNumber(v);
+}
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Maths/missing-number.cpp:38]: and return 0 and the end of the main function to tell that your program runs successfully

**Review by UG-SEP — CHANGES_REQUESTED** [review]
I am really sorry for the late review.
Hope you understand😊
You need to make some changes as I suggested below
**Beside Changes I would like to tell you that you have done a great job an awesome way to present your code. Everything is properly used**

**Comment by UG-SEP** [comment]
> Hi @ming-tsai, No one has reviewed this pull request yet. Will you please assign a different reviewer to this? Thanks.

Really sorry for the late review I was busy with my *CLS* work

**Comment by ming-tsai** [comment]
> > Hi @ming-tsai, No one has reviewed this pull request yet. Will you please assign a different reviewer to this? Thanks.
> 
> Really sorry for the late review I was busy with my _CLS_ work

Ok, don't worry about it

**Comment by samughodake** [comment]
> I am really sorry for the late review. Hope you understand😊 You need to make some changes as I suggested below **Beside Changes I would like to tell you that you have done a great job an awesome way to present your code. Everything is properly used**

It's all okay. I understand.
I will make the desired changes.
Thanks.

**Comment by samughodake** [comment]
Made all the desired changes @UG-SEP

**Review by UG-SEP — APPROVED** [review]
Great Work @samughodake 
Thank you for your valuable contribution 
Hope you contribute more to this project😊

**Pull request merged**